### PR TITLE
seacas: new version

### PR DESCRIPTION
**Pull request by gsjaardema**
Update fmt dependency to support fmt@11:
Use an adios2 release instead of master
New release of seacas

### CODE REVIEW

**Comment by gsjaardema** [comment]
spackbot fix style

File: var/spack/repos/builtin/packages/seacas/package.py
Comment (wdconinc) [inline var/spack/repos/builtin/packages/seacas/package.py:269]: Does this need to be a specific release or is any newer fine?
```suggestion
        depends_on("adios2@2.10.1:")
```

Comment (gsjaardema) [reply var/spack/repos/builtin/packages/seacas/package.py:269]: It is the lastest release that thy currently have, so I'm not sure whether future releases will work.  I do know that there is currently a problem with there latest development version (that they are aware of). 

I'm fine assuming it will work with future, or waiting until they release and then testing...

Comment (wdconinc) [reply var/spack/repos/builtin/packages/seacas/package.py:269]: I am not familiar enough with these packages to know how tightly interwoven they are and how much breakage is typically involved in adios2 version upgrades, so I'll defer to your expertise and mark this as resolved.

**Review by wdconinc — APPROVED** [review]
Thanks!

**Pull request merged**
